### PR TITLE
Clean-up of return types, indentation and misc other things

### PR DESCRIPTION
**Pull request by WasabiFan**
These are mostly style changes to make the codebase consistent. I chose the styles I've seen, used and liked in other projects.

Also, I would like to add `public` wherever it was omitted, but that might be controversial. I like to add it in JS/TS because the default accessibility is the opposite of that in most other languages.

### CODE REVIEW

**Comment by dlech** [comment]
I'm OK with adding `public` wherever it makes sense.

However, I would like to have explicit return types on all functions event if it is `void`.

I've basically been using https://github.com/Microsoft/vscode-extension-samples/tree/master/tree-view-sample as a style reference.

**Comment by WasabiFan** [comment]
> I've basically been using https://github.com/Microsoft/vscode-extension-samples/tree/master/tree-view-sample as a style reference.

I would argue that conforming to the style preferences of a single developer who was asked to write some sample code isn't necessarily a good approach :laughing: I've seen how these samples are produced (I have some of my own in Microsoft products) and I can say rom experience that the style in those files wasn't chosen by any sort of a well-structured process. That doesn't necessarily mean that this guy's style is bad, just that it isn't a good reference for decision making.

**Comment by dlech** [comment]
In this case, the "one guy" contributing to the samples includes several of the core the developers of vscode itself and the style seems to match the style of vscode itself, which is what I am going for here.

**Comment by dlech** [comment]
But, if it makes you feel better, I will revise my previous statement to say that I'm using https://github.com/microsoft/vscode as a style guide.

**Comment by WasabiFan** [comment]
OK, now we have a concrete style reference! I'll update this to match.

File: src/extension.ts [hunk 1]
```diff
@@ -285,7 +285,7 @@ enum DeviceState {
 }
 
 class DeviceTreeItem extends vscode.TreeItem {
-    rootDirectory : File;
+    private rootDirectory : File;
```
Comment (dlech) [inline src/extension.ts]: FYI, this needs to be public. I didn't notice until after I merged and got a compile error.

**Pull request merged**